bot/providers/google_image_analyzer.go

```diff
@@ -24,7 +26,8 @@ func NewGoogleVisionAnalyzer(ctx context.Context, opts ...option.ClientOption) (
 	if err != nil {
 		return nil, fmt.Errorf("failed to create vision client: %w", err)
 	}
-	return &GoogleVisionAnalyzer{client: client}, nil
+	logger, _ := zap.NewProduction()
```

Comment:
@jules this should use the existing logger pattern
```suggestion
	logger := logs.NewZapLogger().Named("google_vision_analyzer")
```